test/test.options.subcommand-hyphen.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Module dependencies.
+ */
+
+var program = require('../')
+    , should = require('should');
+
+program
+    .command('subcommand')
```

Comment:
Could you please set the indent width to 2?

Reply:
@abetomo I fixed it. Thank you for your review!

Reply:
👍